docs/server/installation.rst

```diff
@@ -388,55 +232,23 @@ software. To run it, just issue:
 
    (env) $ pootle start
 
-
```

Comment:
Probably that is done by other people, but I find it preferable to be done this way.